toolkits/global/packages/global-footer/scss/50-components/footer.scss

```diff
@@ -0,0 +1,107 @@
+.c-footer {
+	background-color: $footer--background-color;
+	font-family: $footer--font-family;
+	padding-top: $footer--padding-top;
+	padding-bottom: $footer--padding-bottom;
+	color: $footer--text-color;
+}
+
+.c-footer__container {
+	@include u-container;
+	@include u-hide-print;
+}
+
+.c-footer__links {
+	@include u-list-inline;
+	margin-bottom: $footer--links-margin-bottom;
+	line-height: $footer--item-line-height;
+
+	li:not(:last-child) {
+		margin-right: $footer--item-margin-right;
+	}
+}
+
+.c-footer__legal {
+	color: $footer--text-color;
+	font-size: $footer--font-size;
+	margin-top: $footer--legal-margin-top;
+	margin-bottom: $footer--margin-bottom;
+}
+
+.c-footer__link {
+	@include u-link-inherit;
+	white-space: nowrap;
+
+	&:focus {
+		outline: spacing(4) solid #fc0;
+	}
+
+	> img {
+		vertical-align: middle;
+	}
+}
+
+button.c-footer__link {
+	@include u-link-underline;
+	border: none;
+	padding: 0;
+	background: none;
+	font-family: $footer--font-family;
+}
+
+.c-footer__list {
+	@include u-list-reset();
+}
+
+.c-footer__grid {
+	display: flex;
+	flex-wrap: wrap;
+
+	@supports (display: grid) {
+		display: grid;
+		grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+		grid-column-gap: $footer--spacing-medium;
+		grid-row-gap: spacing(32);
+	}
+}
+
+.c-footer__group {
+	flex: 1 1 50%;
+	max-width: 50%;
+	padding-right: $footer--spacing-medium;
+	margin-bottom: $footer--spacing-medium;
+
+	@include media-query('lg') {
+		flex-basis: 25%;
+		max-width: 25%;
+	}
+
+	@supports (display: grid) {
+		padding-right: 0;
+		max-width: none;
+		margin-bottom: 0;
+	}
+}
+
+.c-footer__group--separator {
```

Comment:
💡
For the "product only footer"(?) bits do we want to expose a fresh variable for Sass to determine wether to compile this code or not?

```
@if variable-exists($product-footer--exists) {
    /* The CSS for the product footer */
}
```

Then folks would need to include `$product-footer--exists: true` in their config.

Reply:
every byte counts :D 

but I think this is a larger architectural and 'how we write CSS' question that could be had which shouldn't block this PR :)